ethcore/types/src/engines/mod.rs

```diff
@@ -92,7 +92,7 @@ pub enum SealingState {
 }
 
 /// The number of generations back that uncles can be.
-pub const MAX_UNCLE_AGE: usize = 6;
+pub const MAX_UNCLE_AGE: u64 = 6;
```

Comment:
Is this change because `block_number` is u64? 

Then I suggest to change [fn find_uncle_headers](https://github.com/paritytech/parity-ethereum/blob/master/ethcore/blockchain/src/blockchain.rs#L1326) take take `u64` instead of `usize`,  in order to avoid truncating casts (unlikely happen but still...)